backup_and_restore/application_backup_and_restore/oadp-rosa/oadp-rosa-backing-up-applications-known-issues.adoc

```diff
@@ -0,0 +1,12 @@
+:_content-type: ASSEMBLY
+[id="oadp-rosa-backing-up-applications-known-issues"]
+= OADP ROSA known issues
+include::_attributes/common-attributes.adoc[]
+:context: oadp-rosa-backing-up-applications-known-issues
+
+
+.Restic is not supported or recommended
+
+* link:https://issues.redhat.com/browse/OADP-1054[CloudStorage: openshift-adp-controller-manager crashloop seg fault with Restic enabled]
+* link:https://issues.redhat.com/browse/OADP-1057[Cloudstorage API: CSI Backup of an app with internal images partially fails with plugin panicked error]
+* (Affects OADP 1.1.x_ only): link:https://issues.redhat.com/browse/OADP-1055[CloudStorage: bucket is removed on CS CR delete, although it doesn't have "oadp.openshift.io/cloudstorage-delete": "true"]
```

Comment:
This issue is also closed, as per the last comment on this issue, it seems it no more reproduced in 1.1. Please cross check with Maya Peretz 
~~~~~~
checked again in 1.1.1, doesn't happen there. Closing the issue then.
~~~~~